lib/jsdom/living/nodes/HTMLOutputElement-impl.js

```diff
@@ -10,19 +10,56 @@ class HTMLOutputElementImpl extends HTMLElementImpl {
   constructor(args, privateData) {
     super(args, privateData);
     this._labels = null;
+    this._defaultValue = null;
+    this._valueMode = "default";
   }
 
   _barredFromConstraintValidationSpecialization() {
     return true;
   }
 
+  _formReset() {
+    if (this._valueMode === "value") {
+      this.textContent = this._defaultValue;
+    }
+
+    this._defaultValue = null;
+    this._valueMode = "default";
+  }
+
+  get type() {
+    return "output";
+  }
+
   get labels() {
     return getLabelsForLabelable(this);
   }
 
   get form() {
     return formOwner(this);
   }
+
+  get value() {
+    return this.textContent;
+  }
+
+  set value(val) {
+    this._valueMode = "value";
+    this._defaultValue = this.textContent;
```

Comment:
I was storing the `defaultValue` to be able to satisfy from the spec.

> The reset algorithm for output elements is to set the element's value mode flag to default and then to set the element's textContent IDL attribute to the value of the element's default value (thus replacing the element's child nodes).

After playing around with the output element it behaved (in chrome) as the `defaultValue` being set to the last `textContent` when `value` is set. 
```
const output = document.createElement('output');
output.textContent = '123';
output.value; // "123"
output.defaultValue; // "123"

output.value = 'abc';
output.value; // "abc"
output.defaultValue; // "123"
``

Looking at this again today I am realizing that this would not work correctly when the value is set a second time because the default value will be overridden. It should probably change to this:

```
  set value(val) {
    if (this._valueMode === "default") {
      this._defaultValue = this.textContent;
    }

    this._valueMode = "value";
    this.textContent = val;
  }
```

I think in either case we need to store the defaultValue when putting the output element into "value" mode in order to properly reset its value on form reset don't we?